genkit-tools/common/src/eval/parser.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import * as _ from 'lodash';
```

Comment:
Let's avoid adding lodash. Its use it increasingly discouraged.
Many of the things can be done with vanilla JS.